keys/keys.go

```diff
@@ -0,0 +1,10 @@
+package keys
```

Comment:
Does this need a copyright header/package comment?

Reply:
I would put it ([similar example](https://github.com/f-secure-foundry/GoKey/blob/master/keys.go))

Reply:
Done.